ts/src/alpaca.ts

```diff
@@ -298,6 +306,299 @@ export default class alpaca extends Exchange {
         return iso;
     }
 
+    async fetchAccountInfo (params = {}) {
+        /**
+         * @method
+         * @name alpaca#fetchAccount
+         * @description fetches the account level information, including the cash balance
+         * @see https://docs.alpaca.markets/reference/getaccount-1
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @returns {response} the account level information - not-parsed into typed object
+         */
+        // {
+        //     "id": "dd87af37-8ccc-48af-b40f-248115a19354",
+        //     "admin_configurations": {},
+        //     "user_configurations": {
+        //       "max_options_trading_level": 0
+        //     },
+        //     "account_number": "PA376Y73B28S",
+        //     "status": "ACTIVE",
+        //     "crypto_status": "ACTIVE",
+        //     "options_approved_level": 2,
+        //     "options_trading_level": 0,
+        //     "currency": "USD",
+        //     "buying_power": "112643.96",
+        //     "regt_buying_power": "112643.96",
+        //     "daytrading_buying_power": "0",
+        //     "effective_buying_power": "112643.96",
+        //     "non_marginable_buying_power": "56208.81",
+        //     "options_buying_power": "56367.24",
+        //     "bod_dtbp": "0",
+        //     "cash": "56208.81",
+        //     "accrued_fees": "0",
+        //     "pending_transfer_in": "0",
+        //     "portfolio_value": "100531.63",
+        //     "pattern_day_trader": false,
+        //     "trading_blocked": false,
+        //     "transfers_blocked": false,
+        //     "account_blocked": false,
+        //     "created_at": "2024-07-04T11:53:19.230206Z",
+        //     "trade_suspended_by_user": false,
+        //     "multiplier": "2",
+        //     "shorting_enabled": true,
+        //     "equity": "100531.63",
+        //     "last_equity": "100004.46",
+        //     "long_market_value": "44322.82",
+        //     "short_market_value": "0",
+        //     "position_market_value": "44322.82",
+        //     "initial_margin": "113.17",
+        //     "maintenance_margin": "67.9",
+        //     "last_maintenance_margin": "67.9",
+        //     "sma": "0",
+        //     "daytrade_count": 0,
+        //     "balance_asof": "2024-07-05",
+        //     "crypto_tier": 1,
+        //     "intraday_adjustments": "0",
+        //     "pending_reg_taf_fees": "0"
+        //   }
+        const response = await this.traderPrivateGetV2Account (params);
+        return response;
+    }
+
+    async fetchOpenPositions (params = {}) {
```

Comment:
we don't have a method called `fetchOpenPositions`, it should be `fetchPositions` and the signature is different

Reply:
My intention here was not to create `fetchPositions`. I have not implemented that. This is really used to build the balance. You will see it returns the unparsed response, that then get translated into balance structure in fetchBalance.

Perhaps my naming convention is the issue?

https://github.com/ccxt/ccxt/blob/ddc2b464129c07aee8418d5d353aedece014d380/ts/src/alpaca.ts#L368-L421

Reply:
See response above - move the fetchOpenPositions functionality into fetchBalance. Cleanest solution.